tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0011_reorder_mvs.py

```diff
@@ -0,0 +1,451 @@
+from dataclasses import dataclass
```

Comment:
We should probably start commenting these as the large piles of SQL make it hard to see what changed. If you do a file level comment Alembic will print it out as part of the migration as well.